### PR TITLE
allti logo

### PR DESCRIPTION
**Pull request by Lyndomen**
## About the PR
 Take thy form from off my door!

## Why / Balance
no

## Technical details
image

## Media
16x16 logo cant hurt you
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**
:cl:
- add: mail

### CODE REVIEW

**Comment by Lyndomen** [comment]
todo: new real logo eventually

**Review by MilonPL — APPROVED** [review]

File: Resources/Textures/Logo/logo.ico
Comment (MilonPL) [inline Resources/Textures/Logo/logo.ico:1]: should be icon.ico

**Review by MilonPL — CHANGES_REQUESTED** [review]
you need to remove or change icon.ico, icon.svg and logo.svg because we're changing the attributions

File: Resources/Textures/Logo/logo.ico
Comment (Lyndomen) [reply Resources/Textures/Logo/logo.ico:1]: real not fake

File: Resources/Textures/Logo/icon-license.txt [hunk 1]
```diff
@@ -1,14 +1,13 @@
-The files below are licensed under CC-BY-SA 4.0 by the author who can be found at https://leonardo-dabepis.tumblr.com/
+The files below are licensed under CC-BY-SA 4.0 by the author who won't be found, but is Lyndomen
 
 icon.ico
-icon.svg
+allti.xcf
```
Comment (MilonPL) [inline Resources/Textures/Logo/icon-license.txt]: what the fuck is allti.xcf

Comment (Lyndomen) [reply Resources/Textures/Logo/icon-license.txt]: Mine

Comment (MilonPL) [reply Resources/Textures/Logo/icon-license.txt]: ok but it's not in the scope of this PR so doesn't need to be included in the license

**Review by MilonPL — APPROVED** [review]
surely nothing can go wrong

**Comment by Lyndomen** [comment]
Merge my pr.gif

**Pull request merged**